appoint-ease/src/App.js

```diff
@@ -17,18 +17,19 @@ import Footer from './Footer';
 import ResetPassword from './ResetPasswordComponent';
 import { createBrowserHistory } from 'history';
 import SearchList from './Search';
+import JournalEntry from './JournalEntry';
 
 const PrivateRoute = ({ element: Element, isLoggedIn, ...rest }) => (
   isLoggedIn ? <Route {...rest} element={<Element />} /> : <Navigate to="/" />
 );
 
 const App = () => {
-  const [isLoggedIn, setIsLoggedIn] = useState(false);
-  const [userRole, setUserRole] = useState('');
+  const [isLoggedIn, setIsLoggedIn] = useState(true);
```

Comment:
When the application starts "isLoggedIn" is false and "userRole" is null. Maybe you forgot to change these two parts after you tested it.